examples/platform/silabs/efr32/wf200/host_if.cpp

```diff
@@ -532,9 +535,30 @@ static void wfx_events_task(void * p_arg)
         {
             if (!retryInProgress)
             {
+                retryInProgress = true;
+                if (wfx_is_sta_provisioned())
+                {
+                    EFR32_LOG("%s: Station is provisioned", __func__);
+                }
+                else
+                    EFR32_LOG("%s: Station not provisioned", __func__); 
+
+                if (!is_commissioned)
+                {
+                    EFR32_LOG("%s: First commissioning, retry after 5 sec", __func__);
+                    if (retryJoin < MAX_JOIN_RETRIES_COUNT)
+                        vTaskDelay(pdMS_TO_TICKS(WLAN_RETRY_TIMER));
+                }
+                else
+                {
+                    EFR32_LOG("%s: Next attempt after %d Sec", __func__,
```

Comment:
This need to be rebased and updated after SiLabs's WIP for restructuration of folder and renaming of EFR32 to SILABS
I would put an halt be on this before their PR's are merged